drivers/Makefile.dep

```diff
@@ -737,3 +737,7 @@ ifneq (,$(filter xbee,$(USEMODULE)))
   USEMODULE += xtimer
   USEMODULE += netif
 endif
+
+ifneq (,$(filter netdev_%,$(USEMODULE)))
```

Comment:
This line and ...

Reply:
I would keep this line anyway. In worst case it doesn't compile any module, but I plan to add some common helpers for `netdev` that might be used by `netdev_tap` too.